WooCommerce/Classes/ViewRelated/Inbox/InboxNoteRow.swift

```diff
@@ -52,6 +63,11 @@ struct InboxNoteRow: View {
                                 Text(action.title)
                             }
                         }
+
+                        Text(Localization.surveyCompleted)
+                            .secondaryBodyStyle()
+                            .renderedIf(surveyCompleted || (viewModel.isSurvey && viewModel.isActioned))
```

Comment:
❓ I'd assume we don't require the API call `markInboxNoteAsActioned` to succeed to update the UI? I think it makes sense because the effort to handle errors is not worth it, but just asking to make sure.

Reply:
Exactly :nod: we update the status internally with the `surveyCompleted` variable which is only local and updated when one of the survey actions is pressed.